tests/ImageSharp.Tests/Common/SimdUtilsTests.cs

```diff
@@ -212,7 +212,7 @@ static void RunTest(string serialized)
             FeatureTestRunner.RunWithHwIntrinsicsFeature(
                 RunTest,
                 count,
-                HwIntrinsics.DisableSSE41);
+                HwIntrinsics.AllowAll);
```

Comment:
Rules are inherited. 

HwIntrinsics.DisableSSE41 will disable anything SSE41 and above . That includes AVX, AVX2 etc.

Reply:
That is super confusing considering this from jit source:
```
// Enable AVX instruction set for wide operations as default. When both AVX and SSE3_4 are set, we will use the most
// capable instruction set available which will prefer AVX over SSE3/4.
```

Further looking into jit initialization step for intrinsics it become clear that avx/avx2 check comes only after sse42. I've actually seen some kind of a tree for intrinsics checks somewhere in disscussions or even issues maybe and can't find it. I'll try restore it or recreate from https://blog.weghos.com/coreclr/CoreCLR/jit/compiler.cpp.html and add it to the docs of the HwIntrinsics enum so this problem won't happen again.

Thanks for taking a look at this!